app/models/offspring.rb

```diff
@@ -3,4 +3,5 @@ class Offspring < ApplicationRecord
   validates :first_name, presence: true, length: { within: 2..60 }
   validates :last_name, presence: true, length: { within: 2..60 }
   validates :user, presence: true
+  has_many :assignments, dependent: :destroy
```

Comment:
Yes, I understand what you say abaout using active and I actually thought of 2 options:
1. using a flag "active" (but you will have N-1 false values and 1 true)
2. using a method that iterates through the assignment dates and get the most recent one